vm/src/obj/objbytes.rs

```diff
@@ -201,3 +203,58 @@ fn bytes_iter(obj: PyBytesRef, _vm: &VirtualMachine) -> PyIteratorValue {
         iterated_obj: obj.into_object(),
     }
 }
+
+fn compare_slice(a: &[u8], b: &[u8]) -> bool {
+    for (i, j) in a.iter().zip(b.iter()) {
+        if i != j {
+            return false;
+        }
+    }
+    return true;
+}
+
+fn compare_vec(a: &Vec<u8>, b: &Vec<u8>) -> bool {
```

Comment:
I suggest to rename this function to `vec_contains`. The word compare is confusing to me.